src/core/dev_settings.py

```diff
@@ -14,7 +14,7 @@
     'utils.middleware.TimeMonitoring',
     'debug_toolbar.middleware.DebugToolbarMiddleware'
 )
-INSTALLED_APPS = ['debug_toolbar']
+INSTALLED_APPS = ['debug_toolbar', 'django_nose']
```

Comment:
Added nose here but it is missing from requirements.txt

Reply:
It is present in `dev_requirements.txt`

Reply:
Ok - we'll need to document this then because my build error'd out and other will too if they're not also installing dev-requirements.

Reply:
`dev_requirements.txt` are installed by default in the docker image and the dev_settings are used by the dockerised Janeway. I imagine the problem is that you haven't rebuilt your image in a while, so would have errored if it was added to `requirements.txt` as well (i.e. it shouldn't error on others if they rebuild)

Reply:
And if someone not using docker also decides to use the dev_settings file?

Reply:
That worflow is not documented so I don't expect that to happen. In any case, once we move the wiki to the docs page, yes we should explain the `dev_requirements.txt` file

Reply:
Okay, just playing devils advocate. TBH, if I wasn't going to use Docker and was looking for a settings file for use locally and saw dev_settings and example_settings I'd go with dev_settings, but maybe that's just me :+1:

Reply:
In that case you would raise an issue and I would tell you to follow the manual :nerd_face: